src/components/auto-complete/auto-complete.tsx

```diff
@@ -4,7 +4,7 @@ import {Popup} from '../../';
 import {ChangeEvent} from '../../types/events';
 import {FormInputProps} from '../../types/forms';
 import {noop} from '../../utils';
-import {CaretDown} from '../drop-down/drop-down-icons';
+// import {CaretDown} from '../drop-down/drop-down-icons';
```

Comment:
I guess we can remove this?

Reply:
yep, thanks, push that, actually it is because @LeonidLevi doesn't use linting in editor